ajax/libs/angular-growl/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "angular-growl",
+  "version": "0.4.0",
+  "description": "growl like notifications for angularJS projects, using bootstrap alert classes",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/marcorinck/angular-growl.git"
+  },
+  "author": {
+    "name": "Marco Rinck"
+  },
```

Comment:
Let's modified to `"author": "Marco Rinck",` like https://github.com/marcorinck/angular-growl/blob/master/bower.json#L2
We's like let author in one line, if the field contain only fill in author's name.

Thanks.